### DISCUSSION TITLE
Dynamically generate a core block via PHP

### DISCUSSION THREAD

**Discussion by dfilter** [WordPress/gutenberg#63455] [Q&A]
For example how could I go about generating a core/image block on the server side of things. The use case being I want the block to populate a custom image gallery using the post's meta. The block lives inside of the template file since it appears on all posts of a specific type (thus the need for the block to be dynamic).

**Comment by stokesman** [comment]
From the title this seems to be about using PHP templates but not so much in the body. If you are using block templates there is the [block bindings](https://developer.wordpress.org/news/2024/02/introducing-block-bindings-part-1-connecting-custom-fields/) feature that seems suited to your use case.
